### DISCUSSION TITLE
Bypass policy

### DISCUSSION THREAD

**Discussion by hnsantu554** [strongswan/strongswan#2468] [Q&A]
I am using ipsec esp for specific ports in linux
Ex: local_ts = dynamic[udp/22800]

Once connection is established ,is it possible to update policy from protected to bypass ? And bypass to protected dynamically on udp port 22800 whenever required?

Please help me , thank you

**Comment by tobiasbrunner** [comment]
I suppose you could install/uninstall bypass policies dynamically (e.g. via `swanctl --install/--uninstall`), they have higher priority than regular IPsec policies.
